mindsdb_native/libs/data_sources/file_ds.py

```diff
@@ -186,16 +188,22 @@ def _getDataIo(self, file):
             data.seek(0)
             first_few_lines = []
             i = 0
+
+            # need to have some templates to deduce a dialect
+            # but it is not a good idea to deduce dialect by header row
+            # data[0]
             for line in data:
                 if line in ['\r\n','\n']:
                     continue
```

Comment:
Not added by this PR, but not exactly sure why this code has a "continue" it should have a break, i.e. this indicates a final line that only contains a newline.

Reply:
You are right, according to RFC spec CSV file couldn't contain empty lines in the middle. Fixed